font-changer.py

```diff
@@ -0,0 +1,29 @@
+import sys
+import re
+import os
+
+file_to_change = "src/index.html"
+
+output_file = "src/test.html"
+
+if len(sys.argv) < 2:
+    os.exit(1)
+    
+font_chosen = sys.argv[1]
+
+text = open(file_to_change).read()  # read the entire file content
+
+
+
+matches = re.findall(r'\"(.+?)\"',text)  # match text between two quotes
+for m in matches:
+    # if m is a font replace it with the font chosen
+    text = text.replace('font-family: \"%s\"' % m, 'font-family: \"%s\"' % font_chosen)
```

Comment:
this would overwrite every customized font to `font_chosen`, it seems like this is too powerful especially if we choose to have different fonts for different types of text

currently, the headers are a custom font and the text in sub divs are set to the default font, so this regex match is able to replace safely but it could break the html if this wasn't the case

Reply:
i think you can do a regex to see if the div is a header and then change font only if it is

Reply:
I guess if we do precision changes, we're still going to have to do them manually.

If it breaks the html it wouldn't compile right?

Reply:
> i think you can do a regex to see if the div is a header and then change font only if it is

you can do this for changes across a certain category